backends/concrete-cuda/implementation/test_and_benchmark/test/test_wop_bootstrap.cpp

```diff
@@ -152,8 +152,9 @@ TEST_P(WopBootstrapTestPrimitives_u64, wop_pbs) {
             lwe_sk_in, result_ct, input_lwe_dimension, &decrypted_message);
         // Round after decryption
         uint64_t decrypted =
-            closest_representable(decrypted_message, 1, p) >> delta_log_lut;
-        uint64_t expected = plaintext >> delta_log;
+            closest_representable(decrypted_message, 1, p_array[i]) >>
+            delta_log_array[i];
+        uint64_t expected = plaintext >> delta_log_array[i];
         EXPECT_EQ(decrypted, expected)
             << " failed at tau " << i << ", repetition " << r
             << ","
```

Comment:
We should modify the current parameter sets in this test to use for example: tau = 1 in the first, tau = 2 in the second and tau = 3 in the third.

Reply:
Hmm actually we should change those parameter sets to use some from the optimizer @pdroalves could you maybe give a hand to @bbarbakadze on this topic?

Reply:
Let's keep the same parameter sets as before for now (with tau = 1).